WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -2,7 +2,7 @@ import SwiftUI
 
 extension Color {
 
-    static var accent: Color {
+    static var posAccent: Color {
```

Comment:
This is the fix that I mentioned above, in a recent PR, I introduced this new property, and it resulted in an issue with the subtle button style that we use for "Write with AI" button:

<img src=https://github.com/user-attachments/assets/9d606438-ad1d-4bf7-8fcb-1c5ad1618416 width=300 />

My understanding of the issue: given that now we had `UIColor.accent` and `Color.accent` both in the app, the `UIKitAttributes.foregroundColor` started using the `Color` one, and ignore it at runtime, so to avoid any other similar issues, I'm renaming this property to make it clearer it's only for POS screens.

Reply:
Thanks for the explanation!